### PR TITLE
Some CSS fixes to OpenMage adminhtml theme

### LINKED ISSUE #2397: New admin theme: date fields to small
### Preconditions (*)

1. OM 19.4.17
2. new admin theme

### Steps to reproduce (*)

1. got to catalog price rules

### Expected result (*)

grid2

### Actual result (*)

grid1

---

Change CSS min-width for `.grid tr.filter .range div.date`?

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

- made columns bit wider
- removed duplicate code from scss
- fixed wrong imports in scss
- added label for OM admin theme

### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#2397

### Manual testing scenarios (*)

1. `npm install -g sass`
2. ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

**Comment by ADDISON74** [comment]
1) It would be useful if each change was accompanied by a screenshot (for the first and last bullets in the list).

2) In the Magento theme, I solved the issue in which the date was not fully visible in the input box (PR #2330). Although you made the columns wider in the OpenMage theme, the issue remained. Below a comparison between themes.

data_range

3) I do not agree that the modified CSS files should be compressed. On the one hand we will get compressed files and others not in the repository and on the other hand, we affect those who were used to uncompressed files in which they made their own changes (not everyone has knowledge of SASS).

**Comment by ADDISON74** [comment]
I tested the last version and found that you solved the issue of the visibility of the date selected in the input box.

date

**Comment by ADDISON74** [comment]
Please analyze whether a change must be made in the case below. The selection should be aligned with the others and have the same width.

price

**Comment by ADDISON74** [comment]
I appreciate that the CSS files are no longer compressed and it allowed me to go through them in GitHub. You should remove "compressed output" from the PR description.

**Comment by ADDISON74** [comment]
This comment seems to be too far from the left margin. There are a comments like this one.

distance

File: skin/adminhtml/default/openmage/scss/_variables.scss [hunk 1]
```diff
@@ -72,6 +72,70 @@ $font_2: Helvetica;
...
+$color_flamingo_approx: #f16221;
+$black_15: rgba(0, 0, 0, 0.15);
+$black_75: rgba(0, 0, 0, 0.75);
+$color_gothic_10_approx: rgba(117, 146, 163, 0.1);
+$color_gothic_25_approx: rgba(117, 146, 163, 0.25);
+
+
+$hue: 205;
+/*
+                           205 = default,
```
Comment (fballiano) [inline skin/adminhtml/default/openmage/scss/_variables.scss:98]: @ADDISON74 is right, why are these comments so far off?

Comment (ADDISON74) [reply skin/adminhtml/default/openmage/scss/_variables.scss:98]: ... they probably ran for fear of the edge on the left

**Comment by fballiano** [comment]
@ADDISON74 do we like this as is it? :-)

**Comment by ADDISON74** [comment]
@fballiano - I have to test it.

**Comment by sreichel** [comment]
> @fballiano - I have to test it.

Just changed datefields and currency select fields. For demostore, please enable notice to test.

**Comment by ADDISON74** [comment]
Please guide me related to these changes:

- "added label for OM admin theme"
- "For demostore, please enable notice to test"

**Comment by sreichel** [comment]
First is just a new label for github.

Somewhere in admin config you can enable "Show demostore notice".

**Comment by ADDISON74** [comment]
I don't have a problem enabling that feature in Backend (Config > General > Design> HTML Head > Display Demo Store Notice = Yes), but let's confirm what are the changes in this PR. At the top there is the notice, the logo on the left and the two elements on the right no longer overlap. Also, if the width of the window is reduced, the elements on the right no longer overlap the menu.

This is before:

before

This is after:

after

Otherwise, they are fine the 2 changes in the grids (date ranges, currency). Also, the CSS files are no longer compressed, which allowed me to follow the changes.

**Comment by fballiano** [comment]
I'll test this as soon as I close the docblock update, cause I've 3000 modified files in my workspace at the moment :-\

**Review by ADDISON74 — APPROVED** [review]

**Review by fballiano — APPROVED** [review]
tested, ok

**Pull request merged**